catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_NIDEM.jsonnet

```diff
@@ -0,0 +1,168 @@
+local id = 'projects/geoscience-aus-cat/assets/NIDEM';
+local subdir = 'geoscience-aus-cat';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local version = '1.0.0';
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  "gee:skip_indexing": true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  version: version,
+  title: 'National Intertidal Digital Elevation Model 25m ' + version,
```

Comment:
This title is the same as the title on the old NIDEM dataset. I suggest making the old dataset's title "'Preview National Intertidal Digital Elevation Model 25m" to distinguish them. (We prefer to change the start of the title to make search results more distinct.)